README.md

```diff
@@ -1,23 +1,72 @@
 
-
 .Net and Xamarin libraries for the AeroGear Mobile Core
 
+
+|                 | Project Info                                                     |
+| --------------- | ---------------------------------------------------------------- |
+| License:        | Apache License, Version 2.0                                      |
+| Build:          | MSBuild                                                           |
+| Documentation:  | TBD							                                     |
+| Issue tracker:  | https://issues.jboss.org/browse/AEROGEAR                         |
+| Mailing lists:  | [aerogear-users](http://aerogear-users.1116366.n5.nabble.com/)   | 
+|                 | [aerogear-dev](https://groups.google.com/forum/#!forum/aerogear) |
+
+
+## Documentation
+
+1. [End User Getting Started Guide](./Documentations/GettingStarted.md)
+2. [SDK architecture](./Documentations/SDKArch.md)
+
+
+## List of SDKs
+
+AeroGear Services SDK consist of set of separate SDKs
+* [Core](./Documentations/Core.md): Common base for all SDKs
+* Auth: Mobile authentication SDK
+
+## License 
+
+ See [LICENSE file](./LICENSE)
+
+## Development
+
+If you would like to help develop AeroGear you can join our [developer's mailing list](https://groups.google.com/forum/#!forum/aerogear), join #aerogear on Freenode, or shout at us on Twitter @aerogears.
+
 ## Generate API Docs
 
 The API docs are generated using [Doxygen](http://www.stack.nl/~dimitri/doxygen/).
 
 1. Install Doxygen following the [instructions](http://www.stack.nl/~dimitri/doxygen/manual/install.html)
 2. Locate the `doxygen` command line tool.
-    * For Mac, if `Doxygen` is installed in `/Applications`, you can find the command line tool in `/Applications/Doxygen.app/Contents/Resources/doxygen`
+	* For Mac, if `Doxygen` is installed in `/Applications`, you can find the command line tool in `/Applications/Doxygen.app/Contents/Resources/doxygen`
 3. To generate the API docs, run the command line tool in the root of the SDK directory:
-    
-    ```bash
-    cd aerogear-xamarin-sdk
-    # on Mac, run this command
-    /Applications/Doxygen.app/Contents/Resources/doxygen Documentations/Doxyfile
-    ```
-
-    The generated files can be found in the [Documentations](./Documentations) directory.
+	
+	```bash
+	cd aerogear-xamarin-sdk
+	# on Mac, run this command
+	/Applications/Doxygen.app/Contents/Resources/doxygen Documentations/Doxyfile
+	```
+
+	The generated files can be found in the [Documentations](./Documentations) directory.
 4. To add more directories for API doc generation, update the `INPUT` configuration in the [Doxygen configuration file](./Documentations/Doxyfile). Use spaces to separate the files or directories.
+
+## Testing
+
+Use Visual Studio for Windows Test Explorer (`Test > Windows > Test Explorer`)  or Visual Studio for Mac (`View > Test`). 
+
+You can run tests also from the Developer Command Line (in Windows).
+
+
+```bash
+msbuild Core\Tests\Core.Tests.csproj /t:Test 
```

Comment:
can we add something for running the android and ios tests as well?

Reply:
It would be nice, but I need to investigate into this further. I can't run the platform tests on Windows Visual Studio. 
And other way around, it's not possible to run non-platform tests from command-line anywhere but Windows.

Reply:
I don't mean to have run them from the command line. I am just saying that we can add some instructions on how to run the platform tests manually: e.g. run the tests as apps on devices, then run the tests from the device.